pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -17,25 +16,66 @@ This section describes how to upgrade your Kommander Management cluster and all
 - [Download][download-binary] and install the latest DKP CLI binary on your computer.
 - Ensure you are on DKP version 2.1 or 2.1.1 and Kubernetes version 1.21.
 - If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned].
-- Review the [Platform Application version updates][platform-apps] that are part of this upgrade.
+- Review the [Platform Application version updates][release-notes] that are part of this upgrade.
+- Download the following air-gapped bundles: [images bundle][images-bundle], [charts bundle][charts-bundle] and [Catalog Application charts bundle][cat-apps-bundle]. **Skip this if you are not running in an air-gapped environment**.  
```

Comment:
```suggestion
- **This step is only necessary when you are running in an air-gapped environment** Download the following air-gapped bundles: [images bundle][images-bundle], [charts bundle][charts-bundle] and [Catalog Application charts bundle][cat-apps-bundle].  
```
I find it more convenient from a user's perspective to tell them the conditions under which they need to conduct a certain step prior to telling them the steps. Otherwise users might read through the steps, download the bundles and only then see they wouldn't have had to do it in the first place. Plus, I ridded the sentence of a negation. 😊

Reply:
Yes, much more concise!